### ISSUE TITLE
License ?

### ISSUE THREAD

**Issue by suprafun** [TimChinenov/BerryPicker#1]
Hello, under what license is this project released under ? Thank-you.

**Comment by TimChinenov** [comment]
This was a homework assignment, feel free to use it unrestricted.

**Comment by suprafun** [comment]
Thank-you, should I consider the project under MIT license ? Sorry I just want to be clear : )

**Comment by TimChinenov** [comment]
Yes, you can consider it to be under MIT license. If you use it, the most I ask is for the appropriate credit. Have fun!
